lxd/project/permissions.go

```diff
@@ -160,6 +160,11 @@ func checkRestrictions(project *api.Project, instances []db.Instance) error {
 	for _, instance := range instances {
 		if instance.Type == instancetype.Container {
 			for key, value := range instance.Config {
+				// First check if the key is a forbidden low-level one.
+				if isContainerLowLevelOptionForbidden(key) {
```

Comment:
I'm assuming there's a later commit fixing this as the issue now shows two different config keys to control this behavior and with a list of restricted config keys larger than that in this file.

Reply:
Yeah the other config key (for VMs) is introduced in a later commit.

I've now also updated the list of restricted keys to match the ones in the issue.